spec/spec_tests/retryable_writes_spec.rb

```diff
@@ -19,7 +19,7 @@
           let(:client) do
             authorized_client_with_retry_writes
           end
-          
```

Comment:
I've noticed a lot of places with small formatting issues like this. After the 2.6.0 release, we should discuss standardizing on some code formatting tool.